src/main/scala/com/ambiata/disorder/PositiveLongSmall.scala

```diff
@@ -0,0 +1,16 @@
+package com.ambiata.disorder
+
+import org.scalacheck._
+import scalaz._, Scalaz._
+
+case class PositiveLongSmall(value: Long)
```

Comment:
It's a matter of convenience when you need a `Long` because that's what your API expects. You can always go `i.toLong` but I thought it would be nice to provide the different combinations.